build.sh

```diff
@@ -35,13 +35,13 @@ case $LINK in
 esac
 
 # Prefer ninja over make, if it is available
-if which ninja 2>&1 >/dev/null; then
-  CMAKE_OPTS="-G Ninja $CMAKE_OPTS"
-  BUILD_FLAGS=
-else
+#if which ninja 2>&1 >/dev/null; then
+#  CMAKE_OPTS="-G Ninja $CMAKE_OPTS"
+#  BUILD_FLAGS=
+#else
   # BUILD_FLAGS="-- -j$(getconf _NPROCESSORS_ONLN)"
   BUILD_FLAGS=
-fi
+#fi
```

Comment:
We prefer to use Ninja, when it's available, but adding an option to force Makefiles seems like a good idea.

Could you add an option for overriding Ninja here?

Reply:
Exactly, ninja is fast, but not everyone uses them. Makefiles are just more widely adapted and a good tool to build challenges individually. I'll make it an option  :+1: